FINAL_COUNTDOWN.md

```diff
@@ -0,0 +1,74 @@
+- [ ] Figure out functionatily of parent re-renders: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L1070)
+- [ ] Ensure `yield`ed components have the correct `parentView`: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L1716)
+- [ ] Ensure new sub-components get correct `parentView`: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L1760)
+- [ ] Components should have access to the viewRegistry via `parentView`: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L1808)
+- [ ] component should rerender when a property is changed during children\'s rendering: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L1856)
+- [ ] Two-way upstream flow of a CP: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#2027)
+
+### Component Helper
+
+- [ ] Handle non-existing static componet: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js#L483)
+- [ ] Handle non-existing dynamic componet: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js#L489)
+- [ ] `hash` invalidation in `#each`: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js#L508)
+- [ ] Prevent positional params collisions with different components: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js#L557)
+- [ ] Ensure positional params do not pullute attributes: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js#L597)
+
+### Class Helper
+- [ ] Should compute when params change: [Failing test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/custom-helper-test.js#L147)
```

Comment:
This is failing because we were calling compute too often. I think https://github.com/emberjs/ember.js/commit/fbfc4046b0213952f9154cbc44348835fec6feeb might have fixed it. Can someone confirm?